Cloud release 24.2.0 - 28 August 2024 (#193)
seqeralabs/docs: platform_versioned_docs/version-24.1/cloud/changelog.mdx

tags: [changelog]
---

> ### v24.2.0_cycle23 Cloud - 28 August 2024

Reviewer: We'll need to update the date here to the actual release date before we publish.

> ### v24.2.0_cycle23 Cloud - 28 August 2024

Author: ```suggestion
### v24.2.0_cycle23 Cloud - 2 September 2024
```

> Fix: [Data Explorer] enable task workdir navigation using Data Explorer in personal workspaces by adding routing

Reviewer: ```suggestion
Fix: [Data Explorer] enable task working directory navigation using Data Explorer in personal workspaces by adding routing
```

> Fix: Remove -/+ increment buttons from numeric input components in Platform, including Compute Environment and Launch Form interfaces

Reviewer: ```suggestion
Fix: Remove -/+ increment buttons from numeric input components in Platform, including compute environment and launch form interfaces
```

> Fix: Hide compute environment variables section actions when component is disabled (previously indicated that env vars could be edited in this state)

Reviewer: ```suggestion
Fix: Hide compute environment variables section actions when component is disabled (previously indicated that environment variables could be edited in this state)
```
